compiler/z/env/OMRCPU.cpp

```diff
@@ -141,8 +141,8 @@ OMR::Z::CPU::isAtLeastOldAPI(OMRProcessorArchitecture p)
       case OMR_PROCESSOR_S390_Z15:
          ans = self()->getSupportsArch(TR::CPU::z15);
          break;
-      case OMR_PROCESSOR_S390_ZNEXT:
```

Comment:
We should add `Z16` but should not delete the `ZNEXT` here.

Reply:
I have fixed it in my latest commit.